build.sh

```diff
@@ -17,8 +17,9 @@ usage()
     echo
 }
 
-if [ $1 == "-?" ]; then
+if [ "$1" == "-?" ]; then
     usage
+    exit 0
```

Comment:
Huh, I guess that is an option. By the way, build-managed.sh does not do anything with the `-?` parameter right now, even though there is a function to print out the usage. Any objection to just merging this as-is, and perhaps fixing that separately?